_posts/2020-06-07-Human-Curation-and-Digital-Datasets.md

```diff
@@ -0,0 +1,213 @@
+---
+title: "Human Curation and Digital Datasets: A Problem in Multiple Parts"
+author: cgy
+date: 2020-06-07
+---
+
+{% assign author = site.data.authors[page.author]%}
+{{author.name}}
+
+ORCID: <a href="https://orcid.org/{{ author.orcid }}" title="{{author.name}}">{{author.orcid}}</a>
+
+One of the difficulties when creating and maintaining a human curated
+database is that humans are by nature error prone. In this article, I
+will explore some of the challenges and difficulties that the curators
+of the IrishGen database have encountered when dealing with ambiguous
+statements in the genealogies, how human error creeps in over time,
+and, finally, how [IrishGen](https://github.com/cyocum/irish-gen)
+attempts to avoid introducing new errors.
+
+This is not a new phenomenon within the area of the early Irish
+genealogies.  Even in the pre-modern period when scribes and redactors
```

Comment:
Decide whether you are adding one space or two after full stops.

Reply:
OK.  Technically, it does not matter as the browser + CSS ultimately controls how much space there is after a period.  For consistency's sake, I will make it two as that is what I was taught in school.